Libraries/ReactIOS/YellowBox.js

```diff
@@ -172,12 +174,23 @@ const WarningRow = ({count, warning, onPress}) => {
 type StackRowProps = { frame: StackFrame };
 const StackRow = ({frame}: StackRowProps) => {
   const Text = require('Text');
-  const fileParts = frame.file.split('/');
+  const {file, lineNumber} = frame;
+  const fileParts = file.split('/');
   const fileName = fileParts[fileParts.length - 1];
+
   return (
-    <Text style={styles.inspectorCountText}>
-      {`${fileName}:${frame.lineNumber}`}
-    </Text>
+    <TouchableOpacity
+      style={styles.openInEditorButton}
+      onPress={() => {
+        fetch(getDevServer().url + 'open-stack-frame', {
```

Comment:
please extract this to a separate `openFileInEditor` function (element inspector has the same code)